charts/prometheus-pingdom-exporter/values.yaml

```diff
@@ -56,6 +56,9 @@ pod:
     # key: "true"
     # example: "false"
 
+existingSecret:
+  name: ""
+
```

Comment:
Imho key should be configurable too

Reply:
Thanks for the feedback, I'm not sure I understand the change I can make here if you could elaborate a little bit? The current secret has four keys:

```
user
password
appId
accountEmail
```

Are you suggesting something where those four keys could be configurable in the values e.g.

```
existingSecret:
  name: ""
  userKey: "user"
  passwordKey: "password"
  appIdKey: "appId"
  accountEmailKey: "accountEmail"
```

Reply:
Depending where your secrets are coming from you might not be able to control what's the key in the secret is.

So it might be good to have it configurable.